docs/sphinx/user-guide/troubleshooting.rst

```diff
@@ -246,3 +246,13 @@ Resource names should always start with ``/v2`` and end with a trailing ``/``.
 following command will add a permission to ``test-user`` to create repositories::
 
     pulp-admin auth permission grant --resource /v2/repositories/ --login test-user -o create 
+
+pulp_workers, pulp_resource_manager, pulp_celerybeat not restarting after upgrade
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When SELinux is Enforcing, the pulp-selinux package is installed. Starting with pulp-selinux-2.5,
```

Comment:
Is pulp-selinux really only installed if the machine is in Enforcing?

Regardless, even after pulp-selinux is installed, if the user switches SELinux to Permissive pulp-selinux is still installed. The first sentence probably should be removed.